codegen/src/generator/ec2.rs

```diff
@@ -398,6 +399,7 @@ if prefix != \"\" {{
 
 fn generate_struct_field_serializers(shape: &Shape) -> String {
     shape.members.as_ref().unwrap().iter().map(|(member_name, member)| {
+        let tag_name = capitalize_first_letter_of_string(member.location_name.as_ref().unwrap_or(member_name));
```

Comment:
(This comment is on a specific commit hash's file change list.  It would probably go away if this commit disappears due to a force push.)